server/sockets/list.js

```diff
@@ -606,6 +606,31 @@ const archiveList = (socket, dashboardClients, cohortClients, listClients) =>
       });
   });
 
+const deleteList = (socket, dashboardClients, cohortClients) =>
+  socket.on(ListActionTypes.DELETE_SUCCESS, data => {
+    const { listId } = data;
+
+    socket.broadcast
+      .to(`sack-${listId}`)
+      .emit(ListActionTypes.DELETE_AND_REDIRECT, data);
+
+    dashboardClients.forEach(client => {
+      const { socketId } = client;
+
+      socket.broadcast
+        .to(socketId)
+        .emit(ListActionTypes.DELETE_SUCCESS, listId);
+    });
+
+    cohortClients.forEach(client => {
```

Comment:
You iterate through all cohortClients and send this event to all regardless of whether they are members of this list or not. You should send this event to list viewers which are on this cohort view